settings.gradle

```diff
@@ -29,28 +29,35 @@ pluginManagement {
 }
 
 rootProject.name = 'motif'
-include 'lib'
```

Comment:
Also did a alphabetization of this list in addition to the `:` to `-` adjustment.